test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Node dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	pressWithModifier,
+	META_KEY,
+	pressTimes,
+	//getEditedPostContent,
+} from '../support/utils';
+
+async function uploadImageInTheMediaLibrary( assetFileName ) {
+	await page.waitForSelector( '.media-modal input[type=file]' );
+	const inputElement = await page.$( '.media-modal input[type=file]' );
+	const testImagePath = path.join( __dirname, '..', 'assets', assetFileName );
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	await page.waitForSelector( `.media-modal li[aria-label="${ filename }"]` );
+}
+
+async function navigateToNextRegion() {
+	await page.keyboard.down( 'Alt' );
```

Comment:
It would be nice to build the following utility to simplify this:
```js
useKeyboardShortcut( 'Alt+Shift+n' )
```

Reply:
We have https://github.com/WordPress/gutenberg/blob/master/test/e2e/specs/block-hierarchy-navigation.test.js#L15

Reply:
I didn't now it can take multiple modifiers :)